adapters/bidder.go

```diff
@@ -141,4 +135,8 @@ func (r *RequestData) SetBasicAuth(username string, password string) {
 	r.Headers.Set("Authorization", "Basic "+base64.StdEncoding.EncodeToString([]byte(username+":"+password)))
 }
 
+type ExtraRequestInfo struct {
+	PbsEntryPoint metrics.RequestType
+}
```

Comment:
Moved from the deleted bidder info file. Kept in the same package with same names.